mod/redir.php

```diff
@@ -36,7 +36,7 @@ function redir_init(App $a) {
 			|| (!local_user() && !remote_user()) // Visitors (not logged in or not remotes) can't authenticate.
 			|| (!empty($a->contact['id']) && $a->contact['id'] == $cid)) // Local user is already authenticated.
 		{
-			System::externalRedirectTo($url != '' ? $url : $contact_url);
+			System::externalRedirect($url != '' ? $url : $contact_url);
```

Comment:
This case can be replaced by `defaults($url, $contact_url)` although I'd much rather wait for us to be able to write `$url ?? $contact_url` when we decprecate PHP 5.

Reply:
btw. http://php.net/supported-versions.php
PHP 5.x is moving fast to EOL

Reply:
I know, Friendica 2018.12 will be the last version supporting PHP 5.